sdk/src/main/scala/com/horizen/account/storage/AccountStateMetadataStorageView.scala

```diff
@@ -256,6 +283,14 @@ class AccountStateMetadataStorageView(storage: Storage) extends AccountStateMeta
       case _ => // do nothing
     }
 
+    receiptsOpt.foreach(receipts => {
+        for (r <- receipts) {
+          val key = r.transactionHash
```

Comment:
let's construct the key using as a `hash( "receipt" | r.transactionHash)` similar to other cases.